### PR TITLE
[MM-93]: Added the test cases for the image in the comment notification.

### PR DESCRIPTION
**Pull request by arush-vashishtha**
### Summary
This PR consists the test cases for the Image attached to a comment in a issue or merge request on Gitlab.

### CODE REVIEW

File: data/test-cases/plugins/gitlab/subscriptions-and-notifications/Image_in_gitlab_comment.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Connect your Gitlab account to MM account.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/gitlab/subscriptions-and-notifications/Image_in_gitlab_comment.md]: Same as previous one.

File: data/test-cases/plugins/gitlab/subscriptions-and-notifications/Image_in_gitlab_comment.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+2. In the desired channel or DM/GM create subscriptions for `issue_comments` and `merge_request_comments` by using the slash command `/gitlab subscriptions add <owner>/<repo>` on MM.
+3. Navigate to the subscribed repo and add comment with an attachment of an image to any desired `issue` or `merge request` on Gitlab.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/gitlab/subscriptions-and-notifications/Image_in_gitlab_comment.md]: repo in gitlab

Comment (AayushChaudhary0001) [reply data/test-cases/plugins/gitlab/subscriptions-and-notifications/Image_in_gitlab_comment.md]: add an image in the comments

File: data/test-cases/plugins/gitlab/subscriptions-and-notifications/Image_in_gitlab_comment.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+4. Navigate to the desired channel or DM/GM on MM.
+
+**Step 2**
+
+1. Connect your Gitlab account to MM account.
+2. In the desired channel or DM/GM create subscriptions for `issue_comments` and `merge_request_comments` by using the slash command `/gitlab subscriptions add <owner>/<repo>` on MM.
+3. Navigate to the subscribed repo and add comment with an embedded image to any desired `issue` or `merge request` on Gitlab.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/gitlab/subscriptions-and-notifications/Image_in_gitlab_comment.md]: same as above

**Review by AayushChaudhary0001 — CHANGES_REQUESTED** [review]

**Review by AayushChaudhary0001 — APPROVED** [review]

**Review by raghavaggarwal2308 — APPROVED** [review]

**Pull request merged**